src/index.html

```diff
@@ -1,17 +1,170 @@
 <!DOCTYPE html>
-<html lang="en">
-
+<html lang="pl">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="title" content="Patrycja Kołaczyńska - architektura" />
+    <meta name="description" content="Patrycja Kołaczyńska - architektura" />
+    <title>Patrycja Kołaczyńska - architektura</title>
+    <script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule="" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;500&family=Rozha+One&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/main.min.css">
-    <title>Document</title>
 </head>
-    <h1>Hi</h1>
 <body>
 
+    <!-- Header with navigation menu -->
+    <header class="heading" class="container">
+        <div class="heading--title" id="heading--title">
+            PATRYCJA KOŁACZYŃSKA
+        </div>
+        <nav>
+            <ul class="heading--navigation">
+                <li><a class="heading--link" href="#about-me">O MNIE</a></li>
+                <li><a class="heading--link" href="#gallery">PROJEKTY</a></li>
+                <li><a class="heading--link" href="#contact">NAPISZ DO MNIE</a></li>
+            </ul>
+        </nav>
+        <h1 class="heading--paragraph">Stworzę dla Ciebie 
+            <div>
+                <span class="heading--active">budynek,</span>
+                <span class="heading--active">wnętrze,</span>
+                <span class="heading--active">koncepcję.</span> 
+            </div>
+        </h1>
+    </header>
+
+    <!-- About me section -->
+    <section class="about-me" id="about-me">
+        <div class="about-me--hi-slider">
+            <div>CZEŚĆ</div>
```

Comment:
tutaj powinno być coś innego niż div, np. h6. Nie sprawdzam, ale pytam - rozumiem, że nie może być sam tekst obok siebie?

Ja bym to zrobiła tak, żeby zachować semantykę 
```
<p class="about-me--hi-slider"><span> CZEŚĆ </span></p>